codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/FluentClientGenerator.kt

```diff
@@ -315,10 +316,38 @@ class FluentClientGenerator(
     }
 
     private fun RustWriter.renderFluentBuilder(operation: OperationShape) {
+        val outputType = symbolProvider.toSymbol(operation.outputShape(model))
+        val errorType = symbolProvider.symbolForOperationError(operation)
         val operationSymbol = symbolProvider.toSymbol(operation)
+
         val input = operation.inputShape(model)
         val baseDerives = symbolProvider.toSymbol(input).expectRustMetadata().derives
         // Filter out any derive that isn't Clone. Then add a Debug derive
+        // input name
+        val fnName = clientOperationFnName(operation, symbolProvider)
+        rustTemplate(
+            """
+                impl #{InputPath} {
+            /// Creates a fluent builder from this builder.
+            pub async fn send_with(self, client: &crate::Client) -> #{Result}<#{OperationOutput}, #{SdkError}<#{OperationError}>>
```

Comment:
To fix some of CI, you'll need to pass in the second generic arg on `SdkError` conditionally based on the `codegenContext.smithyRuntimeMode.defaultToMiddleware` flag. The middleware mode uses `aws_smithy_http::operation::Response`, and the orchestrator mode uses `http::response::Response`. Here's an example: https://github.com/awslabs/smithy-rs/blob/8c3b7cdf5bfce37b0ac985defcf18286812970f7/aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsPresigningDecorator.kt#L282-L286

Then the next problem is that the `crate::Client` sometimes has generics on it... so I think you just need to make it `crate::Client${generics.decl}` (or `generics.inst`? I can't remember 😅 ) to fix that.

Reply:
generics.inst seems to be working! Thanks for the advice!